### PR TITLE
Fix bug where 'startOnDeploy' results in its opposite action

### PR DESCRIPTION
**Pull request by davidstaheli**

### CODE REVIEW

**Review by leantk — APPROVED** [review]

File: Extension/CloudFoundry/task.loc.json [hunk 1]
```diff
@@ -12,8 +12,8 @@
   "author": "Microsoft",
   "demands": [],
   "version": {
-    "Major": 1,
-    "Minor": 123,
+    "Major": 2,
```
Comment (jpricket) [inline Extension/CloudFoundry/task.loc.json]: did you mean to update the major version only in this task? Is this a breaking change?

**Review by jpricket — APPROVED** [review]
Looks good. 
one comment

File: Extension/CloudFoundry/cloudfoundry.ts [hunk 2]
```diff
@@ -144,7 +144,7 @@ function pushAppToCF() {
                 cfPush.arg('--no-route');
             }
 
-            if (tl.getBoolInput('startOnDeploy', false)) {
+            if (!tl.getBoolInput('startOnDeploy', false)) {
```
Comment (jpricket) [inline Extension/CloudFoundry/cloudfoundry.ts:147]: You should probably add some tests to cover this case.

File: Extension/CloudFoundry/task.loc.json (hunk 1 above)
Comment (madhurig) [reply Extension/CloudFoundry/task.loc.json]: I would say don't update the major version. There are very few uses of this task, so impact should be low.

**Review by madhurig — APPROVED** [review]

**Comment by davidstaheli** [comment]
Fixes #11

**Pull request merged**